contributing_to_docs/term_glossary.adoc

```diff
@@ -635,6 +635,24 @@ A service account binds together:
 * a principal that can be authenticated and authorized
 * a set of secrets
 
+''''
+=== single-node OpenShift
+
+Usage: single-node OpenShift
+
+Single-node OpenShift (or {product-title} on a single-node cluster) is a deployment footprint that provides control plane and worker node capabilities in a single server for deployments in constrained environments.
+
+Do not use: Single Node Openshift (SNO).
+
+''''
+=== three-node OpenShift
+
+Usage: three-node OpenShift
+
+Three-node OpenShift is a compact cluster deployment footprint on three nodes for deployments in constrained environments. It provides three control plane nodes that you configure as schedulable for workloads.
+
+Do not use: Three Node Openshift.
```

Comment:
Should "compact cluster" or variations of that be discouraged as well in favor of this new term?

Reply:
Maybe? I don't mind the phrase "compact cluster" as a synonym for three-node cluster. It's in use widely already.

Reply:
@sdodson do you have any more thoughts on this? Is there someone else that should review or comment?

Reply:
Going to merge this, so if there are more thoughts about "compact cluster", we can address in a separate PR.